src/platform/apollolake/include/platform/platform.h

```diff
@@ -104,7 +104,7 @@ struct sof;
 #define PLATFORM_DMA_TIMEOUT	1333
 
 /* DMA host transfer timeouts in microseconds */
-#define PLATFORM_HOST_DMA_TIMEOUT	200
+#define PLATFORM_HOST_DMA_TIMEOUT	250
```

Comment:
is there any guidance from hardware folks and/or from previous implementation, or this is just an experimental fix?
Also you have typos in commit message

Reply:
@mmaka1 @bkokoszx Fwiw, the original values were determined from trace data. i.e. on BYT it was easy to see when host DMA started and completed (which was very consistent) and easy to set a value.

Reply:
@plbossart 
Now it is just experimental value.

Reply:
@bkokoszx From what I understand, the problem is only on APL, so I think changing one header is enough.